pom.xml

```diff
@@ -77,6 +77,8 @@
     <project.findbugs.version>3.0.2</project.findbugs.version>
     <deploy.autorelease>false</deploy.autorelease>
     <project.error-prone.version>2.35.1</project.error-prone.version>
+    <project.protobuf.version>3.25.5</project.protobuf.version>
```

Comment:
Thanks, I will need to double check this because I know CEL brings in Protobuf-Java 4.x. 

I just need to confirm compatibility. I'll coordinate with CEL and confirm.